### ISSUE TITLE
Git authentication error on push

### ISSUE THREAD

**Issue by adam-26** [ForkIssues/Tracker#75]
Hi,

I've have just downloaded Fork, really like the UI and low memory usage :)

However, I'm having a problem trying to push a commit to github, here's what i've done:
 - Opened Fork and signed in with github.com using 2FA successfully
 - Added an existing git repository to Fork that already exists locally (did not use the `clone` feature)
 - When I try to `git push` using Fork, it requests authentication details for github - I provide the correct credentials (verified using the github.com website login to be sure) but Fork returns an Authentication error:

`remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/......`

As previously stated, the credentials I use are correct, i tried using both github username and my registered email address. How can I successfully connect the repository in Fork to github so I can pull/push? I could not find any 'repository settings' or similar menu options to inspect the configuration being used for connecting to github, maybe this would be helpful?

I know i _could_ just clone the repo again, but I'd prefer to avoid having to do that for all my existing repos.

Thanks, Adam.

**Comment by DanPristupov** [comment]
@adam-26 Hi. Thank you for reporting the problem.

Currently you must SSH protocol if you use 2FA, because login/password pair is not enough for 2FA authentication. To switch to SSH you need to update the remote url's for your repositories in the following way:
`https://username@github.com/username/repo` -> `git@github.com:username/repo`
Or just copy and paste the repository address from GitHub. Please let me know if this makes sense.

P.S. Right now I'm working on an authentication approach which allows to use HTTPS authentication even for accounts with 2FA, but it will be available only in the next Fork update.

**Comment by adam-26** [comment]
Thanks for the quick reply.

I copied the SSH address from **my private repo** on github.com and updated the remote address in Fork, now I get the following error:
```
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
I also tried adding a new remote using the SSH from github incase of any internal caching issues, etc... but it made no difference.

Looking at the github 'Fork App' OAuth settings, it has full access to all private repositories. 

Given that I am trying to access a private repository that exists in the same account I used to authenticate to github.com from Fork, I'm not really sure what else to try (I realize the included error message is not really helpful).

Any suggestions?

Thanks again, Adam.
